docs/docs/cheat-sheet.md

```diff
@@ -0,0 +1,283 @@
+
+Get the PDF: <a href="/gatsby-cheat-sheet.pdf" download>gatsby-cheat-sheet.pdf</a>
+
+<figure aria-labelledby="cheat_sheet-text">
+    <h2>Page 1</h2>
+    <a href="/cheat-sheet_page_1.png" title="Click to open image in a new window" target="_blank" style="display:block;">
+        <img src="/cheat-sheet_page_1.png" alt="Cheat Sheet - page 1" style="display:block; margin:0;" />
+    </a>
+    <h2>Page 2</h2>
+    <a href="/cheat-sheet_page_2.png" title="Click to open image in a new window" target="_blank" style="display:block;">
+        <img src="/cheat-sheet_page_2.png" alt="Cheat Sheet - page 2" style="display:block; margin:0;" />
+    </a>
+</figure>
+
+<div id="cheat_sheet-text" style=" position: absolute; height: 1px; width: 1px;overflow: hidden; clip: rect(1px, 1px, 1px, 1px);">
+    <h2>Gatsby Cheat Sheet contents</h2>
+    <p>v1.0 for Gatsby 2.x
+        <a href="https://gatsby.dev/cheatsheet">Latest version <span aria-hidden="true">↗</span></a>
+    </p>
+    <h2>Top Docs</h2>
+    <table>
+        <tbody>
+            <tr>
+                <td>
+                    <p>Gatsby Docs</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/docs">gatsby.dev/docs</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Gatsby on GitHub</p>
+                </td>
+                <td>
+                    <p><a href="https://github.com/gatsbyjs/gatsby">github.com/gatsbyjs/gatsby</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Gatsby Tutorial</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/tutorial">gatsby.dev/tutorial</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Quick Start<br />(for intermediate and advanced developers)</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/quick-start">gatsby.dev/quick-start</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Gatsby Starters</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/starters">gatsby.dev/starters</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Quick Reference Guide</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/recipes">gatsby.dev/recipes</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Adding Images</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/image">gatsbyjs.dev/image</a></p>
```

Comment:
```suggestion
                    <p><a href="https://gatsby.dev/image">gatsby.dev/image</a></p>
```

https://gatsbyjs.dev/image points to a complete different website :/

Reply:
note: it should fixed in the PDF+Images

Reply:
Hi there, we have a bit of a chicken-and-egg problem here, as the page needs to be published for the links in the printed doc to resolve. Once everything is in place, the links should work.

Reply:
Fixed PDF and images in aa4981b ✌️